rendering/fbx.ksy

```diff
@@ -0,0 +1,148 @@
+meta:
+  id: fbx
+  title: FBX 3D model interchange format
+  file-extension: fbx
+  encoding: UTF-8
+  endian: le
+  license: MIT
+
+seq:
+  - id: header
+    type: header
+  - id: records
+    type: node_record
+    repeat: until
+    repeat-until: _.end_offset == 0
+
+types:
+
+  header:
+    seq:
+      - id: magic
+        contents: "Kaydara FBX Binary  \0"
```

Comment:
```yaml
type: strz
valid: "Kaydara FBX Binary "
```

Reply:
I tried this, and I get an error that I don't fully understand.

```
/types/header/seq/0: parsing expression 'Kaydara FBX Binary  ' failed on 1:9, expected "or" | CharsWhile(Set( , n)) | "\\\n" | End
```

Wait, is `valid` expecting an _expression_?

Reply:
How about `'"Kaydara FBX Binary "'`?

Reply:
Works. Same workaround works for the array one as well.